docs/en/ui/README.md

```diff
@@ -81,73 +79,68 @@ templates:
 
 ## Trace Query
-Trace query is a typical feature as SkyWalking provided distributed agents.
+Since SkyWalking provides distributed agents, trace query is a key feature.
 
 <img src="https://skywalking.apache.org/ui-doc/7.0.0/trace.png"/>
 
-* **Trace Segment List** is not the trace list. Every trace has several segments belonging to different services. If  
-query by all services or by trace id, different segments with same trace id could be list there.
-* **Span** is clickable, the detail of each span will pop up on the left side.
-* **Trace Views** provides 3 typical and different usage views to visualize the trace. 
+* **Trace Segment List** is not the same as a trace list. Every trace has several segments belonging to different services. If you start a query by all services or by trace IDs, different segments with the same trace ID may be listed there.
+* **Span** can be clicked. The details of each span will pop up on the left.
+* **Trace Views** provides three typical and different usage views to visualize the trace. 
 
 ## Profile
-Profile is an interaction feature. It provides the method level performance diagnosis. 
+Profile is an interactive feature. It provides method-level performance diagnoses. 
 
-To start the profile analysis, user need to create the profile task
+To start profile analysis, you need to create a profile task:
 
 <img src="https://skywalking.apache.org/ui-doc/7.0.0/profile-create.png" width="440px"/>
 
 1. Select the specific service. 
-1. Set the endpoint name. This endpoint name typically is the operation name of the first span. Find this on the trace 
+1. Set the Endpoint Name. This endpoint name is typically the operation name of the first span. Find this on the trace 
 segment list view.
-1. Monitor time could start right now or from any given future time.
-1. Monitor duration defines the observation time window to find the suitable request to do performance analysis.
-Even the profile add a very limited performance impact to the target system, but it is still an additional load. This duration
-make the impact controllable.
-1. Min duration threshold provides a filter mechanism, if a request of the given endpoint response quickly, it wouldn't be 
-profiled. This could make sure, the profiled data is the expected one.
-1. Max sampling count gives the max dataset of agent will collect. It helps to reduce the memory and network load.
-1. One implicit condition, **in any moment, SkyWalking only accept one profile task for each service**.
-1. Agent could have different settings to control or limit this feature, read document setup for more details.
-1. Not all SkyWalking ecosystem agent supports this feature, java agent from 7.0.0 supports this in default.
-
-Once the profile done, the profiled trace segments would show up. And you could request for analysis for any span.
-Typically, we analysis spans having long self duration, if the span and its children both have long duration, you could choose
```

Comment:
Just checking if this term, **"long self duration"**, is correct? Or is it simply "long duration"?

Reply:
`long self duration` is correct.

Reply:
> `long self duration` is correct.

Thanks @wu-sheng, I reflected this in a new pull request.